controls/raddiagram/diagram-items/features-container-shapes.md

```diff
@@ -242,36 +242,81 @@ Below you can find a list of all __RadDiagramContainerShape__ members that are r
+* __Rotation__: You can rotate only the __RadDiagramContainerShape__. This means that the rotation won't affect the container's children. You can find more information about shapes rotation in [this tutorial]({%slug raddiagrams-features-rotation%}).			
 
-* __Translation__ - you can translate the __ContainerShape__ along with its children.			
+* __Translation__: You can translate the __ContainerShape__ along with its children.			
 
-* __Scaling__ - you can scale only the __ContainerShape__ without affecting its children scale. You can find more information about __RadDiagramItems__ resizing in [this tutorial]({%slug raddiagrams-features-resizing%}).			
+* __Scaling__: You can scale only the __ContainerShape__ without affecting its children scale. You can find more information about __RadDiagramItems__ resizing in [this tutorial]({%slug raddiagrams-features-resizing%}).			
 
-* __Cut and Copy__ - these clipboard operations work only on the __ContainerShape__. The shapes inside the container won't be cut or copied. You can find more information about the clipboard operations supported in the __RadDiagram__ in [this tutorial]({%slug raddiagrams-features-clipboard%}).			
+* __Cut and Copy__: These clipboard operations work only on the __ContainerShape__. The shapes inside the container won't be cut or copied. You can find more information about the clipboard operations supported in the __RadDiagram__ in [this tutorial]({%slug raddiagrams-features-clipboard%}).			
 
 If you do wish to rotate, scale, cut or copy both the container and its children simultaneously, you can do so by dragging a selection rectangle around the container (instead of just clicking-selecting the container). This selection will contain both the container and the children thus allowing you to perform the aforementioned actions on all items at the same time.  
 
 ## Customize the ContainerShape Appearance
 
 You can easily customize the visual appearance of the __RadDiagramContainerShape__ by using the following properties:		
 
-* __Background__ - gets or sets the brush that specifies the __RadDiagramContainerShape__ background color.			
+* __Background__: Gets or sets the brush that specifies the __RadDiagramContainerShape__ background color.			
 
-* __BorderBrush__ - gets or sets the brush that specifies the __RadDiagramContainerShape__ border color.			
+* __BorderBrush__: Gets or sets the brush that specifies the __RadDiagramContainerShape__ border color.			
 
-* __BorderThickness__ - gets or sets the width of the __RadDiagramContainerShape__ outline.			
+* __BorderThickness__: Gets or sets the width of the __RadDiagramContainerShape__ outline.			
 
 >tip You can use the __RadDiagram.ContainerShapeStyle__ property to explicitly apply a style on all __RadDiagramContainerShapes__ in a __RadDiagram__ instance.		  
 
+## Container Service

```

Comment:
Add an introduction sentence that tells what is Container Service and how to reach those methods? If this is a service show the code that gets the service and tell that the methods are part of the service class.